package.json

```diff
@@ -78,7 +78,7 @@
     "morgan": "1.10.0",
     "nyc": "15.1.0",
     "pbkdf2-password": "1.2.1",
-    "supertest": "6.3.0",
+    "supertest": "6.3.4",
```

Comment:
Can we just unpin this (`^`)? I thought we agreed to just unpin all these as we touched them?